### PR TITLE
Support FORCE_COLOR environment variable

### PR DESCRIPTION
**Pull request by alexkuz**
Same as in [chalk](https://github.com/chalk/chalk#chalksupportscolor)

Useful in case of using [nodejs-dashboard](https://github.com/FormidableLabs/nodejs-dashboard/issues/15)

### CODE REVIEW

**Comment by jbucaran** [comment]
@alexkuz I missed this. Thanks. I want to merge this, but I don't understand the relation with nodejs-dashboard. AFIK they're not using clor.

**Comment by alexkuz** [comment]
@jbucaran yes, they not use it, but if you use `nodejs-dashboard` and try to log colored text with `clor`, it wouldn't work since `nodejs-dashboard` patches `process.stdout`, so `isTTY` is not defined. There's an issue with similar problem: https://github.com/FormidableLabs/nodejs-dashboard/issues/15

**Comment by jbucaran** [comment]
@alexkuz Thanks for ELI5! I get it now haha.

I'm still willing to merge this, but I'm just wondering, if nodejs-dashboard patches process.stdout, why won't they define a dummy `isTTY` prop or similar so that they can work with 3rd party color modules instead of having those modules adapt to nodejs-dashboard 🤔

**Comment by alexkuz** [comment]
I don't know, really. Maybe @jasonwilson would be kind to explain it?

Anyway, I guess, this fix could be useful for other cases too, and it would be nice to be synced with `chalk` in this regard.
